src/schema/metadata/BolusCutOffDelayTime.yaml

```diff
@@ -0,0 +1,20 @@
+---
+key_name: BolusCutOffDelayTime
+description: |
+  Duration between the end of the labeling and the start of the bolus cut-off
+  saturation pulse(s), in seconds.
+  This can be a number or array of numbers, of which the values must be
+  non-negative and monotonically increasing, depending on the number of bolus
+  cut-off saturation pulses.
+  For Q2TIPS, only the values for the first and last bolus cut-off saturation
+  pulses are provided.
+  Based on DICOM Tag 0018,925F `ASL Bolus Cut-off Delay Time`.
+anyOf:
+  - type: number
+    Minimum: 0
```

Comment:
I've noticed there's both `minimum` and `Minimum`.

Reply:
That's weird, I am pretty sure that in JSON schema, the keys are case sensitive, and that `minimum` is a key and `Minimum` is not. I am just surprised that we did not run into errors so far.

will fix in https://github.com/bids-standard/bids-validator/pull/1261